### PR TITLE
Update to v0.5.4

### PR DESCRIPTION
**Pull request by JornC**

### CODE REVIEW

**Comment by JornC** [comment]
- Allow span in the html
- Convert `[[asterisk]]` to a span asterisk

**Review by wilkostalknecht — APPROVED** [review]
LGTM

File: atlas-service-cms/src/main/java/nl/overheid/aerius/templates/TextTemplates.java [hunk 1]
```diff
@@ -44,7 +44,8 @@ public void init() throws IOException {
     resourceFiles.forEach(v -> {
       try {
         final Node document = parser.parseReader(new InputStreamReader(new FileInputStream(v)));
-        final String html = renderer.render(document);
+        final String html = renderer.render(document)
+            .replaceAll("\\[\\[asterisk\\]\\]", "<span class=\"text-asterisk\">*</span>");
```
Comment (Hilbrand) [inline atlas-service-cms/src/main/java/nl/overheid/aerius/templates/TextTemplates.java]: If you don't need reg-exp you can just use `replace`

Comment (JornC) [inline atlas-service-cms/src/main/java/nl/overheid/aerius/templates/TextTemplates.java]:
```suggestion
            .replace("[[asterisk]]", "<span class=\"text-asterisk\">*</span>");
```

**Review by Hilbrand — APPROVED** [review]
LGTM

**Pull request merged**